extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -875,6 +872,15 @@ export class AmpA4A extends AMP.BaseElement {
     return true;
   }
 
+  /**
+   * To be overriden by implementing network.
```

Comment:
I'm fine with that. The only benefit I saw in my approach was allowing networks to perform their reset logic before everything is nulled out.